{"name": "test_delete", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 > 1", "trace": "self = <apiauto.test_doit8.Testcaserun8 object at 0x0000027825CA7978>\n\n    @pytest.mark.run(5)\n    def test_delete(self):\n        with allure.step('删除记录'):\n            message4()\n            logging.info('4.按条件删除记录；')\n>       assert 0 > 1\nE       assert 0 > 1\n\napiauto\\test_doit8.py:58: AssertionError"}, "steps": [{"name": "删除记录", "status": "passed", "start": 1685195447169, "stop": 1685195447170}], "attachments": [{"name": "log", "source": "8a0dbcd1-8553-4cfa-9ba3-5e6fad9bdde1-attachment.txt", "type": "text/plain"}], "start": 1685195447169, "stop": 1685195447171, "uuid": "2464ba9c-8b30-4c60-97a8-b85912e45d7a", "historyId": "72b2cb260a7666eefcd9ad597fd9cd6d", "testCaseId": "e5457e8476f8d6c9e2d4f4c1ec648722", "fullName": "apiauto.test_doit8.Testcaserun8#test_delete", "labels": [{"name": "tag", "value": "run(5)"}, {"name": "parentSuite", "value": "apiauto"}, {"name": "suite", "value": "test_doit8"}, {"name": "subSuite", "value": "Testcaserun8"}, {"name": "host", "value": "zhangtaopc"}, {"name": "thread", "value": "11624-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "apiauto.test_doit8"}]}